app/helpers/hold_helper.rb

```diff
@@ -11,12 +11,13 @@ def render_hold_history_field(field_name, field_value)
     # Set nil values to something that will display onscreen
     field_value = "n/a" if field_value.nil? 
 
+    # For foreign key fields, return a link to the record
     if field_name == 'thesis_id' && thesis = Thesis.find_by(id: field_value)
```

Comment:
field_value=...

Reply:
I refactored this one slightly. I'm still not crazy about it, but I found myself going down a refactoring rabbit hole that didn't feel especially productive.

Reply:
I do this this version is clearer than the previous version but I hear you on the rabbit hole.